docs/usage/security-and-permissions.md

```diff
@@ -0,0 +1,56 @@
+# Security and Permissions
+
+## Comparison table
+
+Refer to this table for a overview of the permissions needed for the different ways to run Renovate:
+
+| Permission name   | Renovate hosted app |  Forking Renovate  |
+| ----------------- | :-----------------: | :----------------: |
+| Dependabot alerts |       `read`        |   not applicable   |
+| Administration    |       `read`        |       `read`       |
+| Metadata          |       `read`        |       `read`       |
+| Checks            | `read` and `write`  |   not applicable   |
+| Code              | `read` and `write`  |       `read`       |
+| Commit statuses   | `read` and `write`  | `read` and `write` |
+| Issues            | `read` and `write`  | `read` and `write` |
+| Pull Requests     | `read` and `write`  | `read` and `write` |
```

Comment:
Our current Forking Renovate docs say that it only needs `read` access to the repository, but this seems to be wrong, according to the GitHub permissions overview for the Forking Renovate app.

Forking Renovate needs `write` access to:

- Commit statuses
- Issues
- Pull Requests

> **Benefits**
> 
> Forking Renovate needs only read level access to any repository it runs on.

https://docs.renovatebot.com/getting-started/running/#benefits

Reply:
It's meant to mean read access to the *code*

Reply:
I'll go open a PR to fix the line in the docs.